lib/src/dotenv.dart

```diff
@@ -49,6 +49,14 @@ class DotEnv {
   /// Clear [env]
   void clean() => _envMap.clear();
 
+  String get(String name, {String? fallback}) {
+    final value = maybeGet(name, fallback: fallback);
+    assert(value == null && fallback == null, 'A non-null fallback is required for missing entries');
```

Comment:
I believe this should be !(value == null && fallback == null) instead

Reply:
Good catch. I have applied a simpler fix since maybeGet will only return null if fallback is null.